package.json

```diff
@@ -441,6 +441,7 @@
     ],
     "configuration": [
       {
+        "title": "Azure-Api-Center",
```

Comment:
Better to be `Azure API Center`？ As it will be shown to end user in VS Code settings.

Reply:
sure